client.go

```diff
@@ -20,6 +20,10 @@ type HAProxyClient struct {
 	conn net.Conn
 }
 
+type HAProxy interface {
+	RunCommand(cmd string) (*bytes.Buffer, error)
```

Comment:
@benjic while it would be possible to pass the `net.Conn` reference in, the dial function(and possibly schema) would have to be called on every RunCommand, as the connection is immediately closed following response.